packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.ct.tsx

```diff
@@ -93,6 +98,29 @@ test.describe("Empty screenshots", () => {
   });
 });
 
+test.describe("Truncated options screenshots", () => {
```

Comment:
could you hover on one of the options so we can verify that the hover background covers the whole multiline option?